packages/neuron-wallet/src/services/wallets.ts

```diff
@@ -55,7 +56,7 @@ export default class WalletService {
   }
 
   public create = ({ name, keystore }: { name: string; keystore: Keystore }): Wallet => {
-    const id = walletStore.saveWallet(name, keystore)
+    const id = walletStore.saveWallet({ id: v4(), name, keystore })
```

Comment:
Now that we pass in pre-generated ID, it should always return a valid ID so no need to check `if (id)`, right?

`saveWallet` doesn't have to return the `ID` anymore.

Reply:
Right